<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaDeCadastro</class>
 <widget class="QWidget" name="TelaDeCadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="l_Cadastro">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CADASTRO</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Matricula">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MATRICULA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Nome">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NOME DO ALUNO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Senha">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SENHA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Matricula">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Nome">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Senha">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_Cadastro_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>CURSO</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_DS">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DATA SCIENCE</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_DFS">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DESIGN FULL STACK</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_PFS">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>570</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PROGRAMAÇÃO FULL STACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SAIR">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>630</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(172, 0, 2);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CADASTRAR">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>630</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chiller</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(172, 0, 2);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>CADASTRAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLb_Riuk">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>231</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../PycharmProjects/loginScreen/style/images/Riuk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../PycharmProjects/loginScreen/style/images/Light.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
